<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>884</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1374</width>
    <height>884</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AI Camera Flowline</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ui/images/aiicon.png</normaloff>ui/images/aiicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#Main_Button {
    background-color: rgb(80, 159, 239);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1314</width>
     <height>818</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_navbar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>61</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignTop">
        <widget class="QLabel" name="header_img">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>ui/images/header.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="navbar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>51</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#navbar {
    background-color: #1f2a44;  /* xanh Ä‘en */
    color: white;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="logo_img">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>141</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>141</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>ui/images/bosch_logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Manual_Button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#Manual_Button {
    background-color: rgb(77, 155, 232);
}
</string>
            </property>
            <property name="text">
             <string>Manual Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Instruction_Button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#Instruction_Button {
    background-color: rgb(77, 155, 232);
}
</string>
            </property>
            <property name="text">
             <string>Instruction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="flowline_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>725</width>
              <height>33</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
              <underline>false</underline>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#flowline_name {
    color: white;
    font-weight: bold;
    qproperty-alignment: 'AlignCenter';
}
</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Flowline 07&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name_app">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>211</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>211</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#name_app {
    color: white;
}</string>
            </property>
            <property name="text">
             <string>HcP/MSE3 AI Flowline System</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="stream_video">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#stream_video{
    border: 1px solid rgb(0, 85, 255);
    border-radius: 20px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::RichText</enum>
      </property>
      <property name="pixmap">
       <pixmap>ui/images/test_view.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="IO_status" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>100</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_sent_to_laser" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>271</width>
           <height>31</height>
          </size>
         </property>
         <widget class="QLabel" name="label_sent_to_laser">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>211</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Belt on scale sent to laser (I3.2)</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="status_sent_to_laser">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>0</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#status_sent_to_laser{
    background-color: rgb(255, 0, 0);
    border-radius: 15px;
}
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_sent_to_oildipping" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>281</width>
           <height>31</height>
          </size>
         </property>
         <widget class="QLabel" name="label_sent_to_oil">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Sent to oildipping machine (I7.3)</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="status_sent_to_oil">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>0</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#status_sent_to_oil{
    background-color: rgb(255, 0, 0);
    border-radius: 15px;
}
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_recieve_from_laser" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>291</width>
           <height>31</height>
          </size>
         </property>
         <widget class="QLabel" name="label_receive_from_laser">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>231</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Recieve from laser machine (Q3.3)</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="status_receive_laser">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>0</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#status_receive_laser{
    background-color: rgb(255, 0, 0);
    border-radius: 15px;
}
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_recieve_from_rework" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>271</width>
           <height>31</height>
          </size>
         </property>
         <widget class="QLabel" name="status_rework_key">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>0</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#status_rework_key{
    background-color: rgb(255, 0, 0);
    border-radius: 15px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_rework">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>0</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>211</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Receive from key rework (NO)</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_instruction" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>119</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_infomation">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>321</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#label_infomation{
    background-color: rgb(226, 226, 226);
    color: black;
	border-radius: 5px;
}</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Infomations&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_instructions" native="true">
         <property name="minimumSize">
          <size>
           <width>841</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#widget_instructions  {
    background-color: rgb(226, 226, 226);
    color: black;
	border-radius: 5px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_instruction">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Instructions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="content_instruction">
            <property name="styleSheet">
             <string notr="true">QLabel#content_instruction{
    background-color: rgb(255, 255, 255);
    color: Black;
	border-radius: 6px;
}</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Instruction for troubleshooting&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_confirm">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>41</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#button_confirm {
    color: black;
    /* font-weight: bold; */
    background-color: rgb(0, 170, 255);
    text-align: center; /* KhÃ´ng chÃ­nh thá»©c há»— trá»£ nhÆ°ng Ä‘á»ƒ mÃ´ táº£ Ã½ Ä‘á»‹nh */
}
</string>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1374</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
